src/components/Hero.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import skyline from '../images/skyline.svg'
+
+const Hero = () => {
+  return (
+    <section>
+      <div className="hero-content-container">
+        <h4>A CODE FOR AMERICA BRIGADE</h4>
+        <h1>Making community services work in a digital age</h1>
```

Comment:
Let's change this to be dynamic content passed down from props. Something like `mainHeading`.